src/UIContainer.ts

```diff
@@ -371,6 +388,7 @@ export class UIContainer extends HTMLElement {
         }
 
         this.setUserIdle_();
+        window.addEventListener('keydown', this._onKeyDown);
```

Comment:
Perhaps we should only attach this listener when `this.deviceType === "tv"`? Since a `keydown` listener on `window` may be quite expensive...

(We'll then need to attach or detach this listener whenever the device type changes.)